compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -602,6 +602,19 @@ struct QuantizeSpecialActivation final : public luci::CircleNodeMutableVisitor<v
     set_act_qparam(node, i_scale, i_zp);
   }
 
+  void visit(luci::CircleUnpackOut *node)
+  {
+    auto unpack = loco::must_cast<luci::CircleUnpack *>(node->input());
+    auto input = loco::must_cast<luci::CircleNode *>(unpack->value());
+    auto i_qparam = input->quantparam();
+    assert(i_qparam->scale.size() == 1); // FIX_CALLER_UNLESS
```

Comment:
We don't need to check `i_qparam` is nullptr or not ?

Reply:
Added! Thanks!